src/domUpdates.js

```diff
@@ -1,29 +1,292 @@
+          <div class="destination-details">
+            <h4 class="city">${destinationSelected.destination}</h4>
+            <p class="lodging-cost">Estimated lodging: $${
+              destinationSelected.estimatedLodgingCostPerDay
+            } / night</p>
+            <p class="flight-cost">Estimated roundtrip flight: $${
+              destinationSelected.estimatedFlightCostPerPerson
+            }</p>
+          </div>
+        </div>
+      </section>
+      <section class="trip-form" id="tripForm">
+        <label>Departure:
+          <input type="date" id="start" name="trip"
+          placeholder="${dayjs().format('YYYY-MM-DD')}"
+          min="${dayjs().format('YYYY-MM-DD')}" max="${dayjs()
+      .add(1, 'year')
+      .format('YYYY-MM-DD')}" required>
+        </label>
+        <br>
+        <label>Return:
+          <input type="number" id="duration" name="trip" placeholder="1" min="1" required>
+          </label>
+        <br>
+        <label>Number of travelers:
+          <input type="number" id="travelers" name="trip" placeholder="1" min="1" required>
+        </label>
+        <br>
+        <button class= "submit-button" id="getEstimate">Get trip estimate</button>
+      </section>
+    </div>
+    `;
+    document
+      .getElementById('getEstimate')
+      .addEventListener('click', getTripEstimate);
+  }
+};
+
+// ________________NEED TO MOVE TO SCRIPTS and Pull OUT DOM UPDATE INFO TO KEEP IN DOM
+
+export const getTripEstimate = () => {
+  const tripForm = document.getElementById('tripForm');
+  const tripDate = document.getElementById('start').value;
+  const formattedStartDate = dayjs(tripDate).format('YYYY/MM/DD');
+  const duration = document.getElementById('duration').value;
+  const numOfTravelers = document.getElementById('travelers').value;
+
+  if (formattedStartDate === '' || !duration || !numOfTravelers) {
+    cardsGrid.innerHTML += `
+      <div id="errorAdjacentElement">
+        <br>
+        <p class="error" id="formError">PLEASE FILL OUT ALL FORM DATA BEFORE SUBMISSION</p>
+      <div>  
+    `;
+
+    setTimeout(function () {
+      const errorMessage = document.getElementById('formError');
+      errorMessage.remove(0);
+    }, 5000);
+
+    document
+      .getElementById('getEstimate')
+      .addEventListener('click', getTripEstimate);
+    return;
+  }
+
+  if (!formattedStartDate || duration || numOfTravelers) {
+    hide(tripForm);
+
+    tripDetails = {
+      id: tripsData.length + 1,
+      userID: currentTraveler.id,
+      destinationID: destinationSelected.id,
+      travelers: parseInt(numOfTravelers),
+      date: dayjs(tripDate).format('YYYY/MM/DD'),
+      duration: parseInt(duration),
+      status: 'pending',
+      suggestedActivities: [],
+      tripsDestination: destinationSelected,
+    };
+
+    currentTraveler.stagedTrip = new Trip(tripDetails, destinationsData);
+    currentTraveler.stagedTrip.updateTripProperties();
+    const estimate = currentTraveler.stagedTrip.cost;
```

Comment:
Take a look at the naming of stagedTrip. I'm not immediately sure what that is representing.